src/VisualStudio/IntegrationTest/TestSetup/VisualStudioIntegrationTestSetup.csproj

```diff
@@ -74,6 +74,11 @@
     <ProjectReference Include="..\TestUtilities\VisualStudioIntegrationTestUtilities.csproj">
       <Name>VisualStudioIntegrationTestUtilities</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Test\Utilities\Portable\TestUtilities.csproj">
+      <!-- https://github.com/dotnet/sdk/issues/433 -->
+      <AdditionalProperties>TargetFramework=net461</AdditionalProperties>
+      <Name>TestUtilities</Name>
```

Comment:
:bulb: consider automated placement of the `Name` property:

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/7cf733282f0a76f9235de7ea24cfcd6efbb2cf17/StyleCop.Analyzers/StyleCop.Analyzers.Vsix/StyleCop.Analyzers.Vsix.csproj#L51-L52

Reply:
I saw that example (actually how I figured out how to fix this bug). Decided to keep this minimal for now as I have much bigger changes that are riding on top of this fix. Essentially moving the Roslyn repo completely over to Directory.build.props + new MSBuild file format.